src/jailer/src/env.rs

```diff
@@ -22,8 +22,15 @@ const STDOUT_FILENO: libc::c_int = 1;
 const STDERR_FILENO: libc::c_int = 2;
 
 const DEV_KVM_WITH_NUL: &[u8] = b"/dev/kvm\0";
+const DEV_KVM_MAJOR: u32 = 10;
```

Comment:
Would be great to add a comment specifying that these constants are taken from `www.kernel.org/doc/Documentation/networking/tuntap.txt`

Reply:
See 3a929b9.